mongoose.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 const enabled = require('debug').enabled;
-const url = process.env.TALK_MONGO_URL || 'mongodb://localhost';
+const url = process.env.TALK_MONGO_URL || 'mongodb://localhost/coral-talk';
```

Comment:
My thought was to have a default db, but I can get behind enforcing people to explicitly set or end up writing to /test/